platform/security/encryption/platform-security-encryption-api/src/main/java/ddf/security/encryption/EncryptionService.java

```diff
@@ -18,17 +18,32 @@
 public interface EncryptionService extends PasswordEncryptor {
 
   /**
-   * Decrypts the unwrapped encrypted value
+   * Encrypts the provided value and wraps the result in "ENC(*)" format. Any input that is already
+   * wrapped is assumed to be encrypted and is a no-op.
    *
-   * @param wrappedEncryptedValue
-   * @return
+   * @param plaintextValue any plaintext value.
+   * @return the encrypted value of the provided plaintext wrapped in "ENC(" and ")", or the
+   *     original value if already wrapped.
+   */
+  String encryptValue(String plaintextValue);
```

Comment:
❗️ This service is whitelisted.

I recall we had discussed this out-of-band and you had a simple solution.

Reply:
Yep. The interface has been reverted and for the time being, all we need to do is embed the `encryptValue` method (along with the `ENC` identifying regex) into all callers until this interface can be updated.